client/main/contextmenu/contextmenus.js

```diff
@@ -0,0 +1,101 @@
+
+'use strict';
+
+const $ = require('jquery');
+
+const RibbonSeparator = require('../ribbon/ribbonseparator');
+const RibbonGroup = require('../ribbon/ribbongroup');
+const ContextMenuButton = require('./contextmenubutton');
+
+const getRowMenuItems = function() {
```

Comment:
no biggie, but this might be better called `createRowMenuItems()`, as `getRowMenuItems()` suggests i'll be getting the same ones each time.